.github/workflows/test-sig-provider.yml

```diff
@@ -0,0 +1,107 @@
+on:
+  push:
+    branches:
+      - 'main'
+  pull_request:
+
+name: Test Suite
```

Comment:
Could you please change the test suite name so that it indicates that those tests are for `sig-provider` service